src/test/java/com/uid2/admin/vertx/test/ServiceTestBase.java

```diff
@@ -85,9 +86,11 @@ public abstract class ServiceTestBase {
     @Mock protected KeysetKeyStoreSnapshot keysetKeyProviderSnapshot;
     @Mock protected RotatingKeyAclProvider keyAclProvider;
     @Mock protected RotatingKeysetProvider keysetProvider;
+    @Mock protected RotatingAdminKeysetStore adminKeysetProvider;
     @Mock protected RotatingKeysetKeyStore keysetKeyProvider;
     @Mock protected RotatingClientSideKeypairStore keypairProvider;
     @Mock protected KeysetSnapshot keysetSnapshot;
+    @Mock protected AdminKeysetSnapshot adminKeysetSnapshot;
```

Comment:
we shouldn't be mocking snapshots - it's easy to construct real ones

Reply:
https://atlassian.thetradedesk.com/jira/browse/UID2-1731